documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -209,6 +209,11 @@ EOF
 ----
 <1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
 <2> Specify the VMware data storage MOR, the {rhv-short} storage domain UUID, or the {osp} `volume_type` UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
++
+[NOTE]
+====
+For OVA, the `StorageMap` can map only a single storage, which all the disks from the OVA are associated with, to a storage class at the destination. For this reason, the storage is referred to in the UI as "Dummy storage for provider OVA".
```

Comment:
```suggestion
For OVA, the `StorageMap` can map only a single storage, which all the disks from the OVA are associated with, to a storage class at the destination. For this reason, the storage is referred to in the UI as "Dummy storage for source provider <provider-name>".
```

and for the NetworkMap, we should reflect that there's another option to specify a source network in a similar way to the destination network in the example above:
```yaml
source:
        name: <network_attachment_definition> <X>
        namespace: <network_attachment_definition_namespace> <Y>
        type: multus
```
and say that the namespace can either be specified using the `namespace` property or with a name that looks like `network-namespace/network-name`